Public/Src/FrontEnd/MsBuild.Serialization/ProjectWithPredictions.cs

```diff
@@ -48,6 +48,7 @@ public sealed class ProjectWithPredictions<TPathType>
         public PredictedTargetsToExecute PredictedTargetsToExecute { get; }
 
         /// <nodoc/>
+        [JsonIgnore()]

```

Comment:
>[JsonIgnore()] [](start = 8, length = 14)

Inconsequential, but without this attribute references are duplicated in the serialized format